lib/core/src/Cardano/Wallet/Types/Read/Tx.hs

```diff
@@ -74,11 +74,29 @@ but the type family `Api.ShelleyLedgerEra` performs the conversion.
 -- This transaction can come from any era, old (Byron) and new (Babbage).
 data Tx where
     Tx
-        :: forall era. Api.IsCardanoEra era
+        :: forall era
+        . Api.IsCardanoEra era
         => Api.CardanoEra era
         -> TxEra era
         -> Tx
 
+instance Show Tx where
```

Comment:
Does `deriving Show` work? 🤔 This way, the `Tx era` constructor would be shown as well — this implementation here swallows it silently.

Reply:
It's possible to derive `Show` if you add a `Show (TxEra era)` constraint to the `era` parameter of the `Tx` constructor:
```hs
data Tx where
    Tx
        :: forall era. (Show (TxEra era), Api.IsCardanoEra era)
        => Api.CardanoEra era
        -> TxEra era
        -> Tx
```
Then you can just write:
```hs
deriving instance Show Tx
```